src/catchup/CatchupWork.cpp

```diff
@@ -106,12 +107,27 @@ CatchupWork::alreadyHaveBucketsHistoryArchiveState(uint32_t atCheckpoint) const
 WorkSeqPtr
 CatchupWork::downloadApplyBuckets()
 {
+    // If first ledger we're going to replay is of older version (before shadow
```

Comment:
(related to the other comment: we should probably move this out of the "apply buckets" method)